src/components/charts/ChartsHelper.js

```diff
@@ -0,0 +1,61 @@
+
+const percentageFormatter = showPercentage => v => (showPercentage ? `${v}%` : v);
```

Comment:
should this just be 
`const percentageFormatter = (showPercentage, v) => (showPercentage ? `${v}%` : v);` ?

I've never seen double arrows before